style.css

```diff
@@ -347,3 +354,223 @@ footer {
   font-size: 22px;
   list-style-type: none;
 }
+
+/* ===== Desktop Version ===== */
+
+@media (min-width: 768px) {
+  /* ===== Navbar Section ===== */
+  .navBar {
+    padding: 0 142px 0 130px;
+    align-items: center;
+    height: 72px;
+  }
+
+  .navBtn {
+    display: flex;
+  }
+
+  .navBtn > .fa-bars {
+    display: none;
+  }
+
+  .navList {
+    display: flex;
+    color: #fff;
+    gap: 16px;
+    align-items: center;
+  }
+
+  .navList > li {
+    list-style-type: none;
+    font-size: 16px;
+    font-family: "Inter", sans-serif;
+  }
+
+  /* ===== Section One ===== */
+  .sectionOne {
+    background-image: url("images/Illustration-header.png");
+    background-repeat: no-repeat;
+    background-position: bottom right;
+    background-size: contain;
+    height: 100vh;
+  }
+
+  .aboutParent {
+    display: flex;
+    flex-direction: row-reverse;
+    gap: 65px;
+    padding: 100px 20px;
+    align-items: center;
+    justify-content: start;
+  }
+
+  .about > h1 {
+    font-size: 60px;
+    font-weight: 400;
+  }
+
+  .about > p {
+    font-size: 30px;
+  }
```

Comment:
- [ ] Please make sure this text breaks into the next line at the same point where it breaks in the Figma design and ensure you are using the correct font-size and spacing